dev/release/requirements.txt

```diff
@@ -0,0 +1 @@
+conda-lock
```

Comment:
It feels slightly odd to use a mix of venv and Conda here (I guess it's easier to set up venv though)

Reply:
to be honest this was only added as a utility I could remove the `requirements.txt` and update the `README` to just specify that `conda-lock` must be installed in order to execute

Reply:
Ok, thanks for explaining!

Reply:
Thanks! I've removed the `requirements.txt` file and updated the `README`